test/controllers/IllnessStartDateController.test.ts

```diff
@@ -0,0 +1,208 @@
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY, OK, UNPROCESSABLE_ENTITY } from 'http-status-codes';
+import { after, before } from 'mocha';
+import request from 'supertest';
+
+import 'app/controllers/EvidenceDownloadController';
+import { Appeal } from 'app/models/Appeal';
+import {
+    ENTRY_PAGE_URI,
+    ILLNESS_START_DATE_PAGE_URI
+} from 'app/utils/Paths';
+
+import { createApp } from 'test/ApplicationFactory';
+import { createDefaultAppeal } from 'test/models/AppDataFactory';
+
+const pageHeading: string = 'When did the illness start?';
+const errorSummaryHeading: string = 'There is a problem';
+const invalidStartDayErrorMessage: string = 'You must enter a day';
+const invalidStartMonthErrorMessage: string = 'You must enter a month';
+const invalidStartYearErrorMessage: string = 'You must enter a year';
+const invalidDateErrorMessage: string = 'Enter a real date';
+const invalidDateFutureErrorMessage: string = 'Start date must be today or in the past';
+let initialIllnessReasonFeatureFlag: string | undefined;
+
+describe('IllnessStartDateController', () => {
+
+    before(() => {
+        initialIllnessReasonFeatureFlag = process.env.ILLNESS_REASON_FEATURE_ENABLED;
+    });
+
+    after(() => {
+        process.env.ILLNESS_REASON_FEATURE_ENABLED = initialIllnessReasonFeatureFlag;
+    });
+
+    const appeal = createDefaultAppeal();
```

Comment:
This function creates appeal that has `reasons` pre-populated with other reason is might not what we want. I'd rather have simple object simulating user who just started with illness reason.